--- a/src/webui/webui.c	2012-09-21 09:04:25.000000000 +0200
+++ b/src/webui/webui.c	2012-09-21 10:16:36.631311665 +0200
@@ -859,9 +859,9 @@ webui_init(void)
 
   http_path_add("/stream",  NULL, http_stream,  ACCESS_STREAMING);
 
-  webui_static_content("/static",        "src/webui/static");
-  webui_static_content("/docs",          "docs/html");
-  webui_static_content("/docresources",  "docs/docresources");
+  webui_static_content("/static",        "/usr/share/tvheadend/src/webui/static");
+  webui_static_content("/docs",          "/usr/share/tvheadend/docs/html");
+  webui_static_content("/docresources",  "/usr/share/tvheadend/docs/docresources");
 
   simpleui_start();
   extjs_start();
