--- a/src/webui/extjs.c	2012-10-12 11:32:54.721041531 +0200
+++ b/src/webui/extjs.c	2012-10-12 11:33:48.696361873 +0200
@@ -87,7 +87,7 @@ extjs_root(http_connection_t *hc, const
 
   htsbuf_qprintf(hq, "<html>\n"
 		 "<script type=\"text/javascript\" src=\""EXTJSPATH"/adapter/ext/ext-base.js\"></script>\n"
-		 "<script type=\"text/javascript\" src=\""EXTJSPATH"/ext-all-debug.js\"></script>\n"
+		 "<script type=\"text/javascript\" src=\""EXTJSPATH"/ext-all.js\"></script>\n"
 		 "<link rel=\"stylesheet\" type=\"text/css\" href=\""EXTJSPATH"/resources/css/ext-all.css\">\n"
 		 "<link rel=\"stylesheet\" type=\"text/css\" href=\"static/livegrid/resources/css/ext-ux-livegrid.css\">\n"
 		 "<link rel=\"stylesheet\" type=\"text/css\" href=\"static/app/ext.css\">\n");
